SummerProject/article/urls.py

```diff
@@ -0,0 +1,8 @@
+""" Urls for articles """
+from django.urls import path
+from .views import comment_form, viewing_comments
+
+urlpatterns = [
+    path('<int:article_id>/comment/', comment_form),
```

Comment:
Let's make this more RESTfull by creating new comment with `POST` to `<int:article_id>/comments/` and returning list of existing commends by `GET` to `<int:article_id>/comments/`

Reply:
Done